src/App.css

```diff
@@ -0,0 +1,33 @@
+html {
+  max-width: 70ch;
+  padding: 3em 1em;
+  margin: auto;
+  line-height: 1.75;
+  font-size: 1.25em;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 3em 0 1em;
+}
+p,
+ul,
+ol {
+  margin-bottom: 2em;
+  list-style: none;
+  font-family: sans-serif;
+}
+a {
+  text-decoration: none;
+}
+
+footer {
+  margin-top: 3em;
+  padding-top: 3em;
+  border-top: 1px solid #eaeaea;
```

Comment:
Since we're gonna use theming, it is better not to hardcode colors, it may be hard to track later

Reply:
yes I was about to do that later, now it's just a template how I see file structure